src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -244,6 +244,7 @@ public FetchCommand fetch_() {
             public boolean shallow;
             public Integer timeout;
             public boolean tags = true;
+            public Integer depth = 1;
```

Comment:
Yea i'll just handle nulls inside git-client just incase. In my testing i couldn't couldn't throw anything in that behaved unexpectedly, but you never know when someone decides to take it and run a creative way.